client/db/bolt/upgrades.go

```diff
@@ -0,0 +1,122 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package bolt
+
+import (
+	"fmt"
+
+	"decred.org/dcrdex/dex/encode"
+	"go.etcd.io/bbolt"
+)
+
+const (
+	initialVersion = 0
+
+	// versionedDBVersion is the second version of the database. It versions
+	// the database by persisting the version.
+	versionedDBVersion = 1
+
+	// DBVersion is the latest version of the database that is understood by the
+	// program. Databases with recorded versions higher than this will fail to
+	// open (meaning any upgrades prevent reverting to older software).
+	DBVersion = versionedDBVersion
+)
+
+// upgrade the database to the next version. Each database upgrade function
+// should be keyed by the database version it upgrades.
+
```

Comment:
This whole comment seems misplaced. I'm not sure what the first sentence is describing., It seems the second sentence is describing `upgrades`, but there is an unnecessary space between the comment and definition.